source/js/components/petition/petition.scss

```diff
@@ -6,7 +8,8 @@
 .cta {
   .sticky-cta {
     position: sticky;
-    top: 57px;
+    top: 5rem;
```

Comment:
the header does not appear to have a fixed height, rather being defined by the text in it, so I made this top offset 5rem to match better - without it, the CTA would end up scrolling on top of/underneath the header, which looked really janky.